tests/Moq.Tests/MockOf.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Xunit;
```

Comment:
I feel that instead of adding a new file for this tests you should extend the tests in Moq.Tests\linq\QueryableMocksFixture and extend the tests named 'shouldOneOf....CreateNewMock'. 
A test 'ShouldOneOfWithContructorParameterAndPredicateCreateNewMock ' would be a good additional test that is missing now.

Reply:
You are right @peterdew, when I added the tests I didn't find where place them but I will change now.
Thanks

Reply:
I am reviewing the code and I didn't add support for default constructor with predicate so is not necesary those tests